MudBlazor.Extensions/Components/MudExUploadEdit.razor.cs

```diff
@@ -389,7 +389,10 @@ public RestrictionType ExtensionRestrictionType
     /// The upload requests.
     /// </summary>
     [Parameter, SafeCategory("Data")]
-    public IList<T> UploadRequests { get; set; }
+    public IList<T> UploadRequests { 
+        get => _value;
+        set => _value = value;
+    }
```

Comment:
Why adding Backing field here? not used.

Reply:
_value is the protected value backer inherited from MudFormComponent. It is used internally by that abstract class for validation so needs to be set. The UploadRequest is just a property front for it.